src/guide/essentials/template-refs.md

```diff
@@ -32,7 +33,7 @@ onMounted(() => {
 </template>
 ```
 
-If not using `<script setup>`, make sure to also return the ref from `setup()`:
+Agar siz `<script setup>`dan foydalanmayotgan bo'lsangiz, `setup()` funksiyasidan `ref` ni qaytarayotganingiz(return qilayotganingiz)ga e'tibor bering:
```

Comment:
![image](https://user-images.githubusercontent.com/12369295/172146945-540ccf63-8cdb-4a22-9cf0-fc8f867fe26f.png)
bu yerda qavsdan oldin probel bo'lishi kerak

Reply:
To'g'rilandi.